globalinits.php

```diff
@@ -63,7 +66,11 @@
 	MEDIA_TYPE_HDDVD	=> array('FormatIcon' => 'gfx/iconhddvdpng.png',  'Icon' => 'gfx/hddvd.jpg',  'Banner' => 'gfx/Banner_HDDVD.png'),
 	MEDIA_TYPE_HDDVD_DVD	=> array('FormatIcon' => 'gfx/iconhddvdpng.png',  'Icon' => 'gfx/hddvd.jpg',  'Banner' => 'gfx/Banner_HDDVD_DVD.png'),
 	MEDIA_TYPE_BLURAY	=> array('FormatIcon' => 'gfx/iconbluraypng.png', 'Icon' => 'gfx/bluray.jpg', 'Banner' => 'gfx/Banner_BluRay.png'),
-	MEDIA_TYPE_BLURAY_DVD	=> array('FormatIcon' => 'gfx/iconbluraypng.png', 'Icon' => 'gfx/bluray.jpg', 'Banner' => 'gfx/Banner_BluRayDVD.png')
+	MEDIA_TYPE_BLURAY_DVD	=> array('FormatIcon' => 'gfx/iconbluraypng.png', 'Icon' => 'gfx/bluray.jpg', 'Banner' => 'gfx/Banner_BluRayDVD.png'),
+	MEDIA_TYPE_ULTRAHD	=> array('FormatIcon' => 'gfx/iconultrahd.png', 'Icon' => 'gfx/ultrahd.jpg', 'Banner' => 'gfx/Banner_UltraHD.png'),
+	MEDIA_TYPE_ULTRAHD_BLURAY	=> array('FormatIcon' => 'gfx/iconultrahd.png', 'Icon' => 'gfx/ultrahd.jpg', 'Banner' => 'gfx/Banner_UltraHDBD.png'),
```

Comment:
Small alignment problem
```suggestion
	MEDIA_TYPE_ULTRAHD_BLURAY    => array('FormatIcon' => 'gfx/iconultrahd.png', 'Icon' => 'gfx/ultrahd.jpg', 'Banner' => 'gfx/Banner_UltraHDBD.png'),
```

Reply:
Good spot thanks, I thought I caught them all when I went back through.

Reply:
I have not resolved this on with the suggestion, as it still does not look right. The array keys have become too long I think. I will push a patch with a larger change clearing up the array itself.